src/components/HeaderBar.vue

```diff
@@ -0,0 +1,64 @@
+<script setup>
+    defineProps({
+      msg: {
+        type: String,
+        required: true
+      }
+    })
+    </script>
+    
+    <template>
+      <header class="header-bar">
+        <img alt="logo" class="logo" src="@/assets/logo.svg" />
```

Comment:
We need to consider to replace this logo.svg.

Reply:
Removed the logo.svg file.